src/Controls/src/Core/ImageSource.cs

```diff
@@ -22,7 +22,7 @@ protected ImageSource()
 
 		public virtual bool IsEmpty => false;
 
-		protected CancellationTokenSource CancellationTokenSource
+		private protected CancellationTokenSource CancellationTokenSource
```

Comment:
Is this too breaking of a change? If people are inheriting from ImageSource thesmelves?

Reply:
I thought I would go with this and see what breaks because this whole system is moving into the service/factories. For example, the Android with Glide doesn't use this at all. I am hoping to move away from this. If it really breaks people, then we can add it back.